mantid/hardware-requirements/plan.md

```diff
@@ -0,0 +1,129 @@
+# Estimating hardware requirements for data reduction at ESS
+
+## Scope
+
+### Live reduction
+
+Preliminary/simplified on-the-fly reduction of data and visualization is a key promise of ESS and will be required for all instruments, any time the beam is on and at times probably also with beam of for other calibration work.
+
+### Interactive reduction
+
+The general view is that data at ESS will be so large that users cannot do data reduction on their laptop or desktop PC.
+Therefore DMSC needs to provide resources for all interactive workflows using the Mantid GUI (`MantidPlot` or its successor) for data reduction and visualization.
+This is required during a user visit as well as the time period after the experiment.
+Furthermore, we need to provide access to such resources before an experiment such that users can familiarize themselves with the tools and provided software infrastructure.
+
+- Due to the interactive nature, high CPU use will alternate with shorter or longer idle breaks.
+  However, generally there will be a rather large permanent memory requirement, i.e., clustering interactive user sessions on a single compute node is only possible to a limited extent, and will require nodes with a large amount of memory.
+
+### Script-based reduction
+
+Reduction of data based on a (Python) script.
+This covers auto reduction and batch reduction.
+
+## Prerequisites
+
+### Baseline
+
+To establish a baseline, we need to survey the hardware used for data reduction at existing facilities using Mantid.
+This should include computers at beamlines, laptops and desktop PCs used by instrument scientists and users, and cluster hardware resources.
+
+### From instrument teams
+
+- A similar reduction workflow (in Mantid) at an operating facility (should be ISIS or SNS) that we can use for benchmarking.
+  - Could be several if the instrument has different operating modes.
+- Listing of build-out phases including, for each phase:
```

Comment:
Good approach. You didn't list this in your summary below, but I think it might be better if we get the accelerator information ourselves, not via the instrument teams.

Reply:
What I meant is: The instrument team may tell us "When the accelarator is at 1 MW, we get 1e6 events/s", i.e., they may specify things depending on acceleator power.